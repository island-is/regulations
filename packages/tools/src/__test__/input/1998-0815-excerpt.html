<p>Brot úr hinni kreisílöngu https://www.reglugerd.is/reglugerdir/allar/nr/0815-1998</p>

<p>––––––––</p>

<h3 align="center" style='text-align:center'><span lang="EN-US">12. gr.</span></h3>

<p class="dagsetning" align="center" style='text-align:center'><span lang="EN-US">Aðrar undanþágur.</span></p>

<p style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto; text-align:justify;tab-stops:19.85pt 36.0pt 72.0pt 108.0pt 144.0pt 180.0pt 216.0pt 252.0pt 288.0pt 324.0pt 360.0pt right 389.8pt left 396.0pt'><strong><span lang="IS" style='mso-ansi-language:IS'><span style='mso-tab-count:1'>      </span></span></strong> <span lang="IS" style='mso-ansi-language:IS'>Eftirfarandi ný efni þarf ekki að tilkynna samkvæmt þessari reglugerð:</span></p>

<p style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto; margin-left:19.85pt;text-align:justify;text-indent:-19.85pt;tab-stops:19.85pt 35.45pt right 389.8pt'><span lang="IS" style='mso-ansi-language:IS'>1.<span style='mso-tab-count:1'>   </span> Efni sem er markaðssett innan EES í minna magni en 10 kg á ári á framleiðanda. Markaðssetning má þó ekki eiga sér stað nema framleiðandi eða innflytjandi hafi sent Hollustuvernd ríkisins upplýsingar um eftirtalin atriði:</span></p>

<p style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto; text-align:justify;tab-stops:19.85pt 35.45pt right 389.8pt'><span lang="IS" style='mso-ansi-language:IS'><span style='mso-tab-count:1'>      </span> a)<span style='mso-tab-count:1'> </span> Auðkenni efnis.</span></p>

<p style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto; text-align:justify;tab-stops:19.85pt 35.45pt right 389.8pt'><span lang="IS" style='mso-ansi-language:IS'><span style='mso-tab-count:1'>      </span> b)<span style='mso-tab-count:1'> </span> Upplýsingar varðandi flokkun og merkingu.</span></p>

<p style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto; text-align:justify;tab-stops:19.85pt 35.45pt right 389.8pt'><span lang="IS" style='mso-ansi-language:IS'><span style='mso-tab-count:1'>      </span> c)<span style='mso-tab-count:1'> </span> Áætlað magn efnis sem er markaðssett á Íslandi og innan EES.</span></p>

<p style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto; margin-left:19.85pt;text-align:justify;text-indent:-19.85pt;tab-stops:19.85pt 35.45pt right 389.8pt'><span lang="IS" style='mso-ansi-language:IS'>2.<span style='mso-tab-count:1'>   </span> Fjölliður sem innihalda í bundnu formi minna en 2% af efni sem ekki er skráð í EINECS.</span></p>

<p>––––––––</p>

<p style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto; text-align:justify;tab-stops:19.85pt right 389.8pt'><span lang="IS" style='mso-ansi-language:IS'><span style='mso-tab-count:1'>      </span> &#64257;egar nýtt efni sem framleitt er hér á landi er flutt út til lands utan EES, sem er á listanum í VIII. viðauka, skal framleiðandi upplýsa Hollustuvernd ríkisins um væntanlegan útflutning áður en hann á sér stað. Upplýsingarnar skulu innihalda nafn efnis samkvæmt CAS<a id="_ftnref1" style='mso-footnote-id:ftn1' href="#_ftn1" name="_ftnref1" title=""><span class="MsoFootnoteReference"><span style='vertical-align:baseline;vertical-align:baseline'>1</span></span></a><span class="MsoFootnoteReference"><span style='vertical-align:baseline;vertical-align: baseline'>)</span></span>- og IUPAC<a id="_ftnref2" style='mso-footnote-id:ftn2' href="#_ftn2" name="_ftnref2" title=""><span class="MsoFootnoteReference"><span style='vertical-align:baseline;vertical-align:baseline'>2</span></span></a><span class="MsoFootnoteReference"><span style='vertical-align:baseline;vertical-align: baseline'>)</span></span>-nafnareglunum, byggingarformúlu, CAS-númer<a id="_ftnref3" style='mso-footnote-id:ftn3' href="#_ftn3" name="_ftnref3" title=""><span class="MsoFootnoteReference"><span style='vertical-align:baseline;vertical-align: baseline'>3</span></span></a><span class="MsoFootnoteReference"><span style='vertical-align:baseline;vertical-align:baseline'>)</span></span> og verslunarheiti.</span></p>

<![if !supportFootnotes]><br clear="all" />
<hr align="left" size="1" width="33%" />
<![endif]>

<div style='mso-element:footnote' id="ftn1">
<p class="MsoFootnoteText"><a id="_ftn1" style='mso-footnote-id:ftn1' href="#_ftnref1" name="_ftn1" title=""><span class="MsoFootnoteReference"><span lang="EN-US" style='vertical-align:baseline;vertical-align:baseline'>1)</span></span></a> <span lang="EN-US" style='font-size:8.5pt'>CAS: Chemical Abstract Service - nafn sem gefið er upp í CAS Chemical Registry System.</span></p>
</div>
