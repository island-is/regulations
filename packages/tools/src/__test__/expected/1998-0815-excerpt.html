<p>
Brot
úr
hinni
kreisílöngu
https://www.reglugerd.is/reglugerdir/allar/nr/0815-1998
</p>

<p>
––––––––
</p>

<h3 class="article__title">
12.
gr.
<em class="article__name">Aðrar
undanþágur.</em>
</h3>

<p>
<span data-legacy-indenter="">
      
</span>
Eftirfarandi
ný
efni
þarf
ekki
að
tilkynna
samkvæmt
þessari
reglugerð:
</p>

<p style="margin-left: 19.85pt; text-indent: -19.85pt">
1.
<span data-legacy-indenter="">
   
</span>
Efni
sem
er
markaðssett
innan
EES
í
minna
magni
en
10
kg
á
ári
á
framleiðanda.
Markaðssetning
má
þó
ekki
eiga
sér
stað
nema
framleiðandi
eða
innflytjandi
hafi
sent
Hollustuvernd
ríkisins
upplýsingar
um
eftirtalin
atriði:
</p>

<p>
<span data-legacy-indenter="">
      
</span>
a)
<span data-legacy-indenter="">
 
</span>
Auðkenni
efnis.
</p>

<p>
<span data-legacy-indenter="">
      
</span>
b)
<span data-legacy-indenter="">
 
</span>
Upplýsingar
varðandi
flokkun
og
merkingu.
</p>

<p>
<span data-legacy-indenter="">
      
</span>
c)
<span data-legacy-indenter="">
 
</span>
Áætlað
magn
efnis
sem
er
markaðssett
á
Íslandi
og
innan
EES.
</p>

<p style="margin-left: 19.85pt; text-indent: -19.85pt">
2.
<span data-legacy-indenter="">
   
</span>
Fjölliður
sem
innihalda
í
bundnu
formi
minna
en
2%
af
efni
sem
ekki
er
skráð
í
EINECS.
</p>

<p>
––––––––
</p>

<p>
<span data-legacy-indenter="">
      
</span>
Þegar
nýtt
efni
sem
framleitt
er
hér
á
landi
er
flutt
út
til
lands
utan
EES,
sem
er
á
listanum
í
VIII.
viðauka,
skal
framleiðandi
upplýsa
Hollustuvernd
ríkisins
um
væntanlegan
útflutning
áður
en
hann
á
sér
stað.
Upplýsingarnar
skulu
innihalda
nafn
efnis
samkvæmt
CAS<sup class="footnote-reference"><a id="_ftnref1" href="#_ftn1">1</a>)</sup>-
og
IUPAC<sup class="footnote-reference"><a id="_ftnref2" href="#_ftn2">2</a>)</sup>-nafnareglunum,
byggingarformúlu,
CAS-númer<sup class="footnote-reference"><a id="_ftnref3" href="#_ftn3">3</a>)</sup>
og
verslunarheiti.
</p>
<hr />
<p class="footnote" id="ftn1">
<sup class="footnote__marker"><a id="_ftn1" href="#_ftnref1">1)</a></sup>
CAS:
Chemical
Abstract
Service
-
nafn
sem
gefið
er
upp
í
CAS
Chemical
Registry
System.
</p>
